libcfnet/protocol.c

```diff
@@ -88,3 +89,122 @@ Seq *ProtocolOpenDir(AgentConnection *conn, const char *path)
 
     return seq;
 }
+
+bool ProtocolGet(AgentConnection *conn, const char *remote_path,
+                 const char *local_path, const uint32_t file_size, int perms)
```

Comment:
Just curious, why did you choose `uint32_t`?

Reply:
I was curious about the same. I don't recall these types being used anywhere else in the codebase. And I think `ssize_t` would make more sense here?